devolving_music/lib/song_scores.py

```diff
@@ -70,107 +33,134 @@ def get_scores(self):
             SongScores.update_song_rating(compare, song1, song2)
 
         return self.song_score_dict
+    
+    def get_compare_submission(self, submission_key):
+        key_list = list(self.song_score_dict.keys())
+        key_list.remove(submission_key)
+        # once you have a critical number of comparisons then pull from quality list
```

Comment:
Unsure whether we need to do this maybe it is best to always pull a random submission instead of pulling from quality list because calculating quality songs is relatively expensive